test/e2e/utils/enodespecbuilder.go

```diff
@@ -0,0 +1,28 @@
+package utils
```

Comment:
The  file should be named: enode_spec_builder.go

Reply:
ExternalNodeSpecBuilder is quite simple. If it is used only by vmagent_test.go, probably just put the code there. No strong opinion.

Reply:
The file could be used for other VM tests, so I prefer to keep it in utils.
I followed the same naming conventions as the existing files in test/e2e/utils/.

Reply:
Unfortunately, they are not named correctly. We use "_" to connect words in file names. You can check other files in the source tree. I would appreciate if you rename other files in test/e2e/utils/ too.

Reply:
Done